gui/projectfiledialog.cpp

```diff
@@ -772,6 +772,7 @@ void ProjectFileDialog::addSingleSuppression(const Suppressions::Suppression &su
 void ProjectFileDialog::setSuppressions(const QList<Suppressions::Suppression> &suppressions)
 {
     mUI->mListSuppressions->clear();
+    // NOLINTNEXTLINE(readability-const-return-type) - neeeds to be a copy since it might be a reference to mSuppressions
```

Comment:
Is this really related?

Reply:
What do you mean?

Reply:
Good point. Maybe yet another false positive in this check. Although it is a valid warning and possibly even a false negative on our side.

Reply:
Sounds like the NOLINT should be for performance-unnecessary-copy-initialization?

Reply:
If it would report that warning and we would not suppress it correctly the CI would fail.

Reply:
Yeah, but unmatched NOLINTs don't seem to cause a warning: https://github.com/llvm/llvm-project/issues/69674

Reply:
Yeah - this is unrelated. Will revert.

Reply:
Done.